<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Invited Talks</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-1.1/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-1.1/highlight.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs && document.readyState && document.readyState === "complete") {
   window.setTimeout(function() {
      hljs.initHighlighting();
   }, 0);
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>

<link rel="stylesheet" href="Kfyin.css" type="text/css" />

</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html"></a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Accueil</a>
</li>
<li>
  <a href="about.html">Communications</a>
</li>
<li>
  <a href="committee.html">Comités</a>
</li>
<li>
  <a href="Dates.html">Dates Importantes</a>
</li>
<li>
  <a href="inscription.html">Inscriptions</a>
</li>
<li>
  <a href="partenaires.html">Partenaires-Sponsors</a>
</li>
<li>
  <a href="programme.html">Programme</a>
</li>
<li>
  <a href="Information.html">Informations pratiques</a>
</li>
<li>
  <a href="English1/index.html">English version</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Invited Talks</h1>

</div>


<div id="thomas-gerds-sensitivity-analysis-using-lava-simulation" class="section level2">
<h2>Thomas Gerds: Sensitivity analysis using lava simulation</h2>
<p>In this talk I will introduce the simulation functions of the R-package lava (Klaus K Holst, 2006-2017, <a href="https://github.com/kkholst/lava" class="uri">https://github.com/kkholst/lava</a>). I will demonstrate how to utilize these tools to perform a sensitivity analysis for a given regression analysis. The first step is to simulate data that are alike the real data under the substantive model which was fitted to the real data for the original regression analysis. To do this we specify a structural equation model for the joint distribution of all variables; in addition to the substantive model which describes the relation between outcome and predictors, this requires parametric models for the interrelationships of the predictor variables. Simulated data under the so obtained system of structural equations should be alike the real data. The second step is then to introduce various deviations from the substantive model in order to study the robustness of the results found in the real data. Which deviations are of particular interest depends very much on the subject matter question and are difficult to discuss in great generality. For the sole purpose of illustration I will simulate data that are alike the pbc data of the survival package, and study the sensitivity of a Cox regression analysis to the existence of an unobserved confounder.</p>
</div>
<div id="farouk-nathoo-bayesian-group-sparse-multi-task-regression-for-imaging-genetics" class="section level2">
<h2>Farouk Nathoo: Bayesian Group-Sparse Multi-Task Regression for Imaging Genetics</h2>
<p>Recent advances in technology for brain imaging and high-throughput genotyping have motivated studies examining the influence of genetic variation on brain structure. In this setting, high-dimensional regression for multi-SNP association analysis is challenging as the response variables obtained through brain imaging comprise potentially interlinked endophenotypes, and there is a desire to incorporate a biological group structure among SNPs based on their genetic arrangement. We consider a recently developed approach for the analysis of imaging genetic studies based on penalized regression with regularization based on a group <span class="math inline">\(l_{2,1}\)</span>-norm penalty which encourages sparsity at both the gene and SNP level. While incorporating a number of useful features, a shortcoming of the proposed approach is that it only furnishes a point estimate and techniques for obtaining valid standard errors or interval estimates are not provided. We solve this problem by developing a corresponding Bayesian formulation based on a three-level hierarchical model that allows for full posterior inference using Gibbs sampling. Techniques for the selection of tuning parameters are investigated thoroughly and we make comparisons between cross-validation, fully Bayes, and empirical Bayes approaches for the choice of tuning parameters. Our proposed methodology is investigated using simulation studies and is applied to the analysis of a large dataset collected as part of the Alzheimer’s Disease Neuroimaging Initiative. I will discuss how our MCMC algorithm scales with an increasing number of SNPs, imaging phenotypes, and subjects, and I will also describe extensions of the model for application to brain-wide data and the corresponding development of a spatial model that is currently in progress. Finally, I will introduce the R package ‘bgsmtr’ that implements our model.</p>
</div>
<div id="julie-bertrand-approche-bayesienne-en-pharmacogenetique-avec-jags-et-rstan" class="section level2">
<h2>Julie Bertrand: Approche Bayésienne en pharmacogénétique avec JAGS et RStan</h2>
<div id="resume" class="section level3">
<h3>Résumé</h3>
<p>Les modèles non linéaires à effets mixtes (MNLEM) utilisent les connaissances a priori sur le médicament pour mieux quantifier la réponse moyenne et sa variabilité intra et inter-patient. Leur développement en pharmacogénétique (PG) devrait permettre d’augmenter la puissance de détection des marqueurs génétiques de la variabilité inter-patient. Pour estimer les paramètres de ces MNLEM, l’approche par maximum de vraisemblance, principalement utilisée en pharmacologie quantitative est mise en défaut dans les études PG de criblage à haut débit ou N&lt;&lt;p. Dans ce cadre, nous explorons une approche de régression pénalisée (extension de type LASSO dans saemix [1]) puis des approches Bayésiennes (JAGs [2] et RStan [3]) par une étude de simulation et sur les données de l’essai PECAN ANRS 12154 [4].</p>
</div>
<div id="abstract" class="section level3">
<h3>Abstract</h3>
<p>Nonlinear mixed effect models use the a priori knowledge on the drug to better quantify its average response, intra and inter variability. Their development in pharmacogenetics (PG) is expected to increase the power of detecting the genetic markers of the interpatient variability. The maximum likelihood approach, commonly used in quantitative pharmacology, fail to estimate these model parameters in high throughput PG studies where N&lt;&lt;p. In that context, we explore a penalized regression (LASSO extension to saemix [1] and Bayesian approaches through a simulation study and on real data from the PECAN ANRS 12154 trial [4].</p>
<p>[1] <a href="https://cran.r-project.org/web/packages/saemix/index.html" class="uri">https://cran.r-project.org/web/packages/saemix/index.html</a></p>
<p>[2] <a href="http://mcmc-jags.sourceforge.net/" class="uri">http://mcmc-jags.sourceforge.net/</a></p>
<p>[3] <a href="http://mc-stan.org/interfaces/rstan" class="uri">http://mc-stan.org/interfaces/rstan</a></p>
<p>[4] Bertrand J, Chou M, Richardson DM, Verstuyft C, Leger PD, Mentré F, et al. Multiple genetic variants predict steady-state nevirapine clearance in HIV-infected Cambodians. Pharmacogenet Genomics. dec 2012;22(12):868‑76.</p>
</div>
</div>
<div id="marie-pierre-etienne-some-statistical-approaches-for-movement-ecology" class="section level2">
<h2>Marie-Pierre Etienne: Some statistical approaches for movement ecology</h2>
<div id="resume-quelques-approches-statistiques-pour-lecologie-du-deplacement" class="section level3">
<h3>Résumé : Quelques approches statistiques pour l’écologie du déplacement</h3>
<p>L’inférence de modèles de mouvement donne un éclairage pertinent sur les mécanismes écologiques responsables des dynamiques au niveau individuel et/ou populationnel. Ces analyses sont essentielles pour les gestionnaires de la faune sauvage pour comprendre les comportements complexes des animaux. Dans le domaine de l’halieutique, Identifier les motifs qui expliquent l’utilisation que les pêcheurs font de l’espace est un élément essentiel d’une gestion durable des ressources marines. Ces deux domaines développent aujourd’hui de grands campagnes de déploiement de GPS. Les questions écologiques que l’ont peut étudier grâce à l’analyse de ce type de données sont très larges et la spécifité des données de trajectoires( objet spatio temporel) réclame de nouveaux développements méthodologiques. Le domaine des statistiques pour l’écologie du mouvement est donc en plein essor et de nombreuses approches différentes sont proposées pour extraire des informations pertinentes de l’analyse des trajectoires. Parmi celles-ci nous pouvons distinguer - des modèles discret en temps et en espace (essentiellement des modèles de Markov sur grille) - des modèles à temps discrets mais à espace d’états continu (marche aléatoire, modèle auto regressif vectoriel et modèles de Markov cachés) - des modèles continus en temps et en espace (mouvement Brownien, Processus de Ornstein Uhlenbeck et plus généralement équation différentielle stochastique)</p>
<p>Dans cet exposé, nous mettrons en avant des développements récents relevant des deux derniers points. En supposant que le mouvement est caractéristique d’un type d’activité, on peut chercher à identifier la succesion des activités en s’intéressant aux changements dans les propriétés du mouvement. Des méthodes de segmentations peuvent être utilisées pour réussir à identifier ces instants de changements. Une autre hypothèse classique consiste à supposer que la partie déterministe d’un mouvement est dirigée par le potentiel de l’environnement. Etudier la trajectoire doit ainsi donner de l’information sur ce potentiel environnemental. Le lien entre trajectoire et environnement est étudié grâce à des modèles d’équations différentielles stochastiques.</p>
</div>
<div id="abstract-some-statistical-approaches-for-movement-ecology" class="section level3">
<h3>Abstract : Some statistical approaches for movement ecology</h3>
<p>Statistical inference of movement models provides many insights on the ecological features that explain population-level dynamics. These analyzes are crucial to wildlife managers to understand complex animal behaviours . In fisheries science, understanding the underlying patterns responsible for spatial use of the ocean is a key aspect of a sustainable management. Both fields promote now large programs to deploy Global Positioning System (GPS) device. The ecological questions addressed by analyzing those GPS data are very broad and the specifity of trajectories data (a spatio temporal object) requires the development of new statistical approaches. The field of statistics for movement ecology is growing and many different approaches have been proposed. Among them, we can distinguish between - discrete time and discrete space approaches (mostly Markov model on lattice), - discrete time and continuous space approaches (Random walk, Vectorial auto-regressiv model, Hidden Markov model, ….) - continuous time and continuous space methods (Brownian motion, Ornstein Uhlenbeck process and more generally Stochastic differential equestions)</p>
<p>In this talk, we will highlight some recent development in the two last cases. - Assuming that the unknown activity of the animal is responsible of some characteristics of the movement, the succession of different activities might be identified by looking for changes in the properties of the trajectory. Change points detection methods can be used for the identification of the activity. - Another common assumption proposes to link the movement to some potential of the environment . Studying the trajectory should therefore provide information on this potential. This link is studied, in a continuous time and continuous space framework, using the Stochastic differential framework.</p>
</div>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
